src/main/java/com/lpvs/service/LPVSLoginCheckService.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2023 Basaeng, kyudori, hwan5180, quswjdgma83
+ *
+ * Use of this source code is governed by a MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package com.lpvs.service;
+
+import com.lpvs.entity.LPVSMember;
+import com.lpvs.exception.LoginFailedException;
+import com.lpvs.repository.LPVSMemberRepository;
+import com.lpvs.repository.LPVSPullRequestRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+@Service
+@Slf4j
```

Comment:
LPVSLoginCheckService.log is not used

Reply:
import lombok.extern.slf4j.Slf4j;
@slf4j

can be removed

Reply:
Done.